### PR TITLE
1747 show shouts

### LINKED ISSUE #1747: 🚀 [Feature] Article's Shout displayed in Profile or not
## :rocket: Feature
If somebody shouts an article it should be possible to decide, if this shout should be also displayed in the profile or not.

### Additional context
This is a privacy option. Somebody could just shout non-political articles for example so that nobody could guess his political direction from it.

### PR DESCRIPTION
**Pull request by Kachulio1**
## 🍰 Pull request
Allows the user to set shouts either public or private

### Issues
- fixes #1747 

### Todo
- [X] None

### CODE REVIEW

**Comment by alina-beck** [comment]
Hey @Kachulio1! Matt said you want some frontend support on this? Do you have any implementation details in mind already?

I think we would need to:
- [ ] add this to the settings page
- [ ] toggle the `shouts` tab on the user profile depending on the setting
- [ ] display a list of shouted posts when the `shouts` tab is selected

Did I miss anything?

**Comment by Kachulio1** [comment]
@alina-beck sorry have been tied up with personal things. sorry for the late response.
should we sync on Monday about things I should work on the backend?

**Comment by alina-beck** [comment]
Hey @Kachulio1, no problem! I think it's done, actually. We already did fetch `shouted posts` because we had the feature before, so I just needed to toggle them on the profile. And the `mutation` you set up worked fine. :+1:

File: backend/src/schema/resolvers/users.js [hunk 1]
```diff
@@ -177,6 +177,7 @@ export default {
         'termsAndConditionsAgreedVersion',
         'termsAndConditionsAgreedAt',
         'allowEmbedIframes',
+        'showShoutsPublicly',
```
Comment (mattwr18) [inline backend/src/schema/resolvers/users.js]: Very important that we not forget to add this! Great job catching this issue before it bites us @kachulio

**Pull request merged**